ci/setup_linux_environment.sh

```diff
@@ -2,14 +2,27 @@
 
 set -e
 
+ARCH=`arch`
+
+if [[ "$ARCH" = "ppc64le" ]]; then
+apt install gnupg2 apt-transport-https -y
+echo "deb https://oplab9.parqtec.unicamp.br/pub/repository/debian/ ./" | sudo tee /etc/apt/sources.list.d/bazel_ppc64le.list
+curl --silent  https://oplab9.parqtec.unicamp.br/pub/key/openpower-gpgkey-public.asc  -o openpower-gpgkey-public.asc
+apt-key add openpower-gpgkey-public.asc
+rm -f openpower-gpgkey-public.asc 
+apt update	
+
+else
+
 echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
 curl --silent https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
 sudo apt-get update -o Dir::Etc::sourcelist="sources.list.d/bazel.list" \
-    -o Dir::Etc::sourceparts="-" \
-    -o APT::Get::List-Cleanup="0"
-sudo apt-get install openjdk-8-jdk bazel
-sudo apt-get install software-properties-common
+-o Dir::Etc::sourceparts="-" \
+-o APT::Get::List-Cleanup="0"
+
+fi
 
-sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
-sudo apt-get install cmake
+sudo apt-get install openjdk-8-jdk bazel -y
```

Comment:
Why do we need a JDK for compiling C++ project? Why do we need bazel?

the build seems to be failing trying to download or install cassandra, why is it needed?

Reply:
Hi,

I was wondering the same, but when I tried looking into this PR (https://github.com/opentracing/opentracing-cpp/pull/90/) it looks like a bazel CI was being targeted to be setup.

I will be more than happy to resubmit and realign the build/test process per the documentation.

Reply:
Ah, I didn't realize this repo uses bazel, and I guess that explains the JDK, but not Cassandra.

Reply:
I still cannot figure out how - Cassandra came into the picture of nowhere, 
but I guess this is not getting introduced by this PR.

Let me know if I should bump up the ubuntu version from trusty to xenial and give it a try, to see if the issue persists?